Google.PowerShell/BigQuery/BqJob.cs

```diff
@@ -0,0 +1,132 @@
+﻿// Copyright 2015-2017 Google Inc. All Rights Reserved.
+// Licensed under the Apache License Version 2.0.
+
+using Google.Apis.Bigquery.v2;
+using Google.Apis.Bigquery.v2.Data;
+using Google.PowerShell.Common;
+using System;
+using System.Net;
+using System.Management.Automation;
+
+namespace Google.PowerShell.BigQuery
+{
+    /// <summary>
+    /// <para type="synopsis">
+    /// Lists all jobs that you started in the specified project or returns information about a specific job.
+    /// </para>
+    /// <para type="description">
+    /// If no Job is specified though ID parameter or object via pipeline, this cmdlet lists all jobs that 
+    /// you started in the specified project. If a Job is specified, it will return a descriptor object 
+    /// for that job. Listing requires Can View or Is Owner project roles. Viewing information about a 
```

Comment:
I took those permissions directly off of the BQ Jobs API page.  I do see that they're Legacy permissions and that the IAM roles are the way to go.  Should I keep these roles to be consistent with the docs or should I put the new IAM roles?  

(I sent feedback on the page mentioning this as well.  Is that read at all, or should I file a bug on the page for being out of date?)